.github/workflows/mvn-branch-ci.yaml

```diff
@@ -0,0 +1,113 @@
+name: Build Branch
+
+# This will be triggered on all PRs and pushes to non-master branches
```

Comment:
would something like
```
on: [push] 
```
work for running this workflow for all branches, including master.

Reply:
Will change this to work for master - going to use the `"**"` syntax (see https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags) so that it is straightforward to modify in the future if needed